pkgs/os-specific/cygwin/w32api/default.nix

```diff
@@ -32,6 +32,8 @@
     };
 
     meta = {
+      # Marked broken 2025-11-28 because it has failed on Hydra for at least one year.
```

Comment:
This broke eval of everything on cygwin.  master already had a fix for this by limiting the platform to cygwin.  I'd like to remove the broken flag in #468157.

Reply:
Noted; looks like 9dd0669e491ee8ce9064ab94a8301b189eabf075 post-dates my PR, which would explain why I missed it :).